src/app/components/LiveLabel/index.tsx

```diff
@@ -0,0 +1,50 @@
+/** @jsx jsx */
+/** @jsxRuntime classic */
+import { PropsWithChildren, useContext } from 'react';
+import { jsx } from '@emotion/react';
+import VisuallyHiddenText from '../VisuallyHiddenText';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import styles from './index.styles';
+import Text from '../Text';
+
+interface LiveLabelProps {
+  ariaHidden?: boolean;
+  liveText?: string;
+  offScreenText?: string;
+  lang?: string;
+  id?: string;
+}
+
+const LiveLabel = ({
+  ariaHidden = false,
+  liveText = 'LIVE',
+  offScreenText = '',
+  lang = 'en-GB',
+  id = '',
```

Comment:
This should not have a default value (you can see in the snapshots that id="". If the id does not have a default value, then it will just be undefined, and will not appear in the snapshots 
```suggestion
  id,
```